doc/sphinxext/related_software.py

```diff
@@ -30,8 +30,10 @@
 # If it's available on PyPI, add it to this set:
 PYPI_PACKAGES = {
     "alphaCSC",
+    "meegkit",
     "meggie",
     "niseq",
+    "pycrostates",
```

Comment:
It looks like we actually don't need it here, either

https://app.circleci.com/pipelines/github/mne-tools/mne-python/25808/workflows/eb3c3881-0351-4947-8aeb-ce7ee7a8a11f/jobs/69774

```suggestion
```